chicagorequests/__init__.py

```diff
@@ -169,6 +174,12 @@ def main(
     intervals = day_intervals(start_datetime, end_datetime)
 
     downloader = Downloader(request_type=request_type)
+    json_file = None
+    if save_json:
+        timestamp = int(datetime.datetime.now().timestamp())
+        filename = f'chicagorequests_{timestamp}.json'
+        print(f'Saving file as {filename}')
```

Comment:
if you want this to get to the user, then his should be:

```python
click.echo(f'Saving file as {filename}', err=True)
```

Reply:
Works for me.